website/docs/wallet/withdraw-validator.md

```diff
@@ -143,22 +142,20 @@ pip3 install -r requirements.txt
 
 ### Step 2: Get your validator withdrawal credentials ready
 
-For this step, you will also need to retrieve your validator’s **withdrawal credentials** from Ethereum. You can retrieve this these credentials from block explorers such as [https://beaconcha.in](https://beaconcha.in). Search for your validator, then navigate to the “deposit” tab in that page, and you’ll find the withdrawal credentials near the bottom of the page and you can copy the value from there.
+For this step, you will also need to retrieve your validator’s **withdrawal_credentials** from Ethereum. 
+you can find the `withdrawal_credentials` for each associated validator in your original deposit_data-XXX.json file when you first used the launchpad.
```

Comment:
Should we add a query for the beacon API to get the credentials?

Reply:
it's right below 
```
curl -X 'GET' \
  'http://YOUR_PRYSM_NODE_HOST:3500/eth/v1/beacon/states/head/validators/YOUR_VALIDATOR_INDEX_OR_PUBLIC_KEY' \
  -H 'accept: application/json'
```